src/main/java/net/comments/sample/TrainingRoomUnlimited.java

```diff
@@ -7,26 +7,21 @@
 
 public class TrainingRoomUnlimited implements Room {
     private Teacher someTeacher;
-    private List<Person> enteredPerson;
+    private final List<Person> enteredPerson;
 
-    public TrainingRoomUnlimited(Teacher someTeacher) {
-        this.someTeacher = someTeacher;
+    public TrainingRoomUnlimited() {
         enteredPerson = new ArrayList<Person>();
 
     }
 
-    // Allows to come into the room. Only one teacher can be in the room.
+    // Allows to come into the room.
```

Comment:
Please use javadoc instead of comments:
```java
/**
* Allows to come into the room.
*/
public void accept(Person somePerson) {
...
}
```

The comment is relevant for all code comments.